repository/src/main/java/org/apache/atlas/discovery/EntityLineageService.java

```diff
@@ -368,7 +368,7 @@ private List<AtlasEdge> getQualifyingProcessEdges(Iterator<AtlasEdge> processEdg
         List<AtlasEdge> qualifyingEdges = new ArrayList<>();
         while (processEdges.hasNext()) {
             AtlasEdge processEdge = processEdges.next();
-            if (lineageContext.evaluate(processEdge.getInVertex())) {
+            if (GraphHelper.getStatus(processEdge) == AtlasEntity.Status.ACTIVE && lineageContext.evaluate(processEdge.getInVertex())) {
```

Comment:
Let's use isAllowDeletedProcess flag instead of this, we already this flag at another place

Reply:
Added isAllowDeletedProcess flag check with ACTIVE edge check